src/web/src/average.ecpp

```diff
@@ -178,19 +180,19 @@ bool database_ready;
         checked_type.assign (AVG_TYPES[0]);
     }
     else if (!is_average_type_supported (checked_type.c_str ())) {
+        std::string expected = TRANSLATE_ME (std::string ("one of the following values: [").append (utils::join (AVG_TYPES, AVG_TYPES_SIZE, ", ")).append("].").c_str ());
```

Comment:
This will also require change as it will result in key/en_us translation actually containing
std::string (" ...